src/govuk/components/cookie-banner/cookie-banner.yaml

```diff
@@ -72,11 +72,11 @@ params:
     - name: hidden
       type: boolean
       required: false
-      description: Defaults to `false`. If you set it to `true`, the message is hidden. You can use `hidden` for client-side implementations where the replacement message HTML is present, but hidden on the page.
+      description: Defaults to `false`. If you set it to `true`, the message is hidden. You can use `hidden` for client-side implementations where the confirmaton message HTML is present, but hidden on the page.
     - name: role
       type: string
       required: false
-      description: Set `role` to `alert` on replacement messages to allow assistive tech to automatically read the message. You will also need to move focus to the replacement message using JavaScript you have written yourself.
+      description: Set `role` to `alert` on confirmaton messages to allow assistive tech to automatically read the message. You will also need to move focus to the confirmaton message using JavaScript you have written yourself.
```

Comment:
I think you meant 'confirmation' here, not 'confirmaton'. Applies to other parts of the file too

Reply:
D'oh, apologies. Thanks Vanita!

Reply:
```suggestion
      description: Set `role` to `alert` on confirmation messages to allow assistive tech to automatically read the message. You will also need to move focus to the confirmation message using JavaScript you have written yourself.
```